src/common/options/rbd-mirror.yaml.in

```
@@ -2,6 +2,13 @@
 ---
 
 options:
+- name: rbd_mirror_remote_osd_op_timeout
+  type: uint
+  level: advanced
+  desc: timeout for remote operations handled by osds such as write (0 is unlimited)
+  default: 5
```

Let's default to the watch timeout:
```suggestion
  default: 30
```